.travis.yml

```diff
@@ -1,7 +1,6 @@
 branches:
   only:
     - master
-    - v2-master
```

Comment:
@gsnavin this should stay in order for Travis CI to keep being triggered on PR targeting v2-master